src/test/run-pass/syntax-extension-source-utils-files/includeme.fragment

```diff
@@ -2,6 +2,6 @@
 
 {
     assert!(file!().ends_with("includeme.fragment"));
-    assert!(line!() == 5u);
+    assert!(line!() == 5_u32);
```

Comment:
I don't mind it either, but I prefer consistency and we don't usually use the `_` in the compiler